profiles/ar71xx-generic.profiles

```diff
@@ -1,13 +1,15 @@
 archer-c7-v1
 archer-c7-v2
 archer-c7-v4
-archer-c7-v5
```

Comment:
Why are you removing the archer-c7-v5 profile?

Reply:
seen, that you corrected this by a follow-up commit. 

This is a perfect use-case for ```git rebase```